index.html

```diff
@@ -6,10 +6,70 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
-
+    <link rel="stylesheet" href="style.css">
 <body>
+    <header class="mainheader">
+        <div class="topheader">
+            <h3 class="topheader1">LOGO</h3>
+            <h3 class="topheader2">CONTACT</h3>
+        </div>
+        <nav class="navbar">
+            <ul class="navbarlist">
+                <li>HOME</li>
+                <li>CAKES</li>
+                <li>ORDERING</li>
+                <li>LESSONS</li>
+                <li>ABOUT</li>
+            </ul>
+        </nav>
+    </header>
+    <section class="boxes">
+        <article class="box-text">
+            <p class="text1">HELLO <br>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>  
+            <p class="text2"> INFORMATION <br> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.</p>
+        </article>
+        <article class="box-image">
+            <figure class="main-img">
+                <p> IMAGE CAROUSEL </p>   
+            </figure>
+            <figure class="carousel-moreimages">
+                <ul class="carousel-moreimages-ul">
+                    <li>IMG 1</li>
+                    <li>IMG 2</li>
+                    <li>IMG 3</li>
+                    <li>IMG 4</li>
+                </ul>
+            </figure>
+        </article>
+    </section>
+    <section class="cake-wrapper-row">
+        <div class="cake-wrapper-row-part">
+            <figure class="wrapperimage">
+                <div class="cakeswrapper">CAKE1</div>
+                <div class="cakesdetails">Lorem ipsum dolor sit amet, consectetur adipiscing elit</div> 
+            </figure>
+            <figure class="wrapperimage">
+                <div class="cakeswrapper">CAKE2</div>
+                <div class="cakesdetails">Lorem ipsum dolor sit amet, consectetur adipiscing elit</div>
```

Comment:
You could use `<figcaption>` for this since you have nested it inside a `figure`

Reply:
Changed to figcaption. Learned another thing, thanks Lucy. You are a star.